k8s/crd-samples/pool-resource-cluster-1.yaml

```diff
@@ -1,13 +1,17 @@
 apiVersion: openebs.io/v1alpha1
 kind: CStorPool
 metadata:
-  name: pool1
+  name: pool1-abc
   node: node-host-label
+  labels:
+    "kubernetes.io/hostname": "node-host-label"
+  finalizers: ["openebs"]
```

Comment:
Please explain the motive behind this finalizer. Which components looks out for this?
Does the finalizer use appropriate naming convention.
IMO "openebs" is not the right term that should be used in this context.

Reply:
CRD watcher looks for it while kubectl delete happens. Can you pls suggest a new term instead of "openebs"?

Reply:
This needs to be worked out in the design doc... Lets solve this there.

Reply:
Since this is WIP and we have decided to move them to openebs/elves, I'll address the review comments in a separate PR. @kmova  @AmitKumarDas